deploy/examples/create-external-cluster-resources.py

```diff
@@ -771,6 +777,25 @@ def get_active_and_standby_mgrs(self):
         # if monitoring endpoint port is not set, put a default mon port
         if not monitoring_endpoint_port:
             monitoring_endpoint_port = self.DEFAULT_MONITORING_ENDPOINT_PORT
+            if self._arg_parser.v2_port_enable:
```

Comment:
v2 port is used for mon, not mgr.

You should add this code in `get_ceph_external_mon_data` and create a function out of this code if needed

Reply:
I was wondering the same, but in this [comment](https://github.com/rook/rook/issues/12297#issuecomment-1570032112) on the issue, you mentioned about the `DEFAULT_MONITORING_ENDPOINT_PORT` which is used in mgrs, hence the respective changes are done. If, we want the changes in the `get_ceph_external_mon_data`, I will update the code accordingly.

Reply:
Updated the code.